minorminer/layout/layout.py

```diff
@@ -306,7 +305,19 @@ def dnx_layout(G, dim=None, center=None, scale=None, **kwargs):
     elif family == "zephyr":
         dnx_layout = dnx.zephyr_layout(
             G, dim=dim, center=dnx_center, scale=dnx_scale)
-
+    
+    # if the output of dnx_layout is not what it should be (at the moment there is a bug in dnx.zephyr_layout and dnx.pegasus_layout)    
```

Comment:
should link to github issue, if we have one; if not we should create it

Reply:
I have a PR for it [here](https://github.com/dwavesystems/dwave-networkx/pull/236). Should also create an issue for it?

Reply:
Just created the issue [here](https://github.com/dwavesystems/dwave-networkx/issues/239). There is also a PR to resolve it [here.](https://github.com/dwavesystems/dwave-networkx/pull/236)

Reply:
Done in commit efb48f1.